src/include/esp/esp_private.h

```diff
@@ -550,16 +550,17 @@ typedef struct {
 #define RECV_LEN()                          recv_buff.len
 #define RECV_IDX(index)                     recv_buff.data[index]
 
-#define ESP_AT_PORT_SEND_BEGIN()            ESP_AT_PORT_SEND_STR("AT")
-#define ESP_AT_PORT_SEND_END()              ESP_AT_PORT_SEND_STR(CRLF)
+#define ESP_AT_PORT_SEND_BEGIN()            ESP_AT_PORT_SEND_CONST_STR("AT")
+#define ESP_AT_PORT_SEND_END()              ESP_AT_PORT_SEND_CONST_STR(CRLF)

```

Comment:
Please replace this line with `ESP_AT_PORT_SEND(CRLF, CRLF_LEN)`. It will be more clear and bug-free if `CRLF` macro later becomes something else. Then I merge. Thanks for contribution.